app/models/payment.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+# Model for a Payment on the Fine page
+class Payment
+  attr_reader :record
+
+  def initialize(record)
+    @record = record
+  end
+
+  def key
+    record['billNumber']
+  end
+
+  def bill_description
+    record['billReasonDescription']
+  end
+
+  def bill_amount
+    record['amount']
+  end
+
+  def bill_date
+    Time.strptime(record['dateBilled'], '%Y-%m-%d')
+  end
+
+  def fee_pay_info
+    record['feePaymentInfo']
+  end
+
+  def fee_item_info
+    record['feeItemInfo']
+  end
+
+  def item_title
+    fee_item_info && fee_item_info['title'] || 'No item associated with this payment'
+  end
+
+  def item_library
+    fee_item_info && fee_item_info['itemLibraryID']
+  end
+
+  def payment_amount
+    fee_pay_info && fee_pay_info['paymentAmount']
+  end
+
+  def payment_date
+    fee_pay_info && Time.strptime(fee_pay_info['paymentDate'], '%Y-%m-%d')
+  end
+
+  def resolution
+    fee_pay_info['paymentTypeDescription'] || payment_amount && 'Paid' || 'Unknown'
+  end
+
+  def paid_fee
+    resolution.match(/credit or debit card/) || false
+  end
+
+  def paid_fee?
+    paid_fee.to_s.eql?('credit or debit card')
```

Comment:
It looks like `paid_fee` is only consumed by this `paid_fee?`.  It seems a bit odd that we maybe casting `false` to a string and then doing an `eql?` check.  Could we just use the existence of a match data to determine if it's a paid fee or not?